src/man/sssd.conf.5.xml

```diff
@@ -3554,12 +3568,6 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                             default the third one is introduced to allow easy
                             integration of users from Windows domains.
                         </para>
```

Comment:
Hi,

I think it might be worth to spell out the character restrictions explicitly here as well. E.g, with respect to group names with an `@` in the name. To make this work a fully-qualified name has to be used since we do not allow `@` in short names. If a user wants to use short name with an `@` character they have to create their own expression.

bye,
Sumit

Reply:
Good Idea. Man page updated.